src/ol_infrastructure/substructure/keycloak/__main__.py

```diff
@@ -274,6 +274,103 @@
 )
 """
 
+# Create OL Data Platform Realm
+ol_data_platform_realm = keycloak.Realm(
+    "ol-data-platform",
+    access_code_lifespan="30m",
+    access_code_lifespan_user_action="15m",
+    attributes={
+        "business_unit": f"operations-{env_name}",
+    },
+    display_name="OL Data",
+    display_name_html="<b>OL Data</b>",
+    enabled=True,
+    login_theme="keycloak",
+    duplicate_emails_allowed=False,
+    otp_policy=keycloak.RealmOtpPolicyArgs(
+        algorithm="HmacSHA256",
+        digits=6,
+        initial_counter=2,
+        look_ahead_window=1,
+        period=30,
+        type="totp",
+    ),
+    realm="ol-data-platform",
```

Comment:
The only thing I want to think/discuss more about is whether to explicitly tie this to the data platform, or consider it to be a bit broader in scope/purpose.

Reply:
I thought about that and felt that it's important with the data involved to be very explicit about this particular realm and have it be very clear.

Reply:
I agree and that makes sense. I just don't want to do unnecessary proliferation of realms if we can avoid it. It definitely makes sense to keep this one narrowly scoped.